CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.5)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Apparently yes, at least on my setup (lxplus)! I am still not 100% sure but I think Geant's cmake configuration exports `--std=c++11` flags without this line.